spec/adapter/postgres_spec.cr

```diff
@@ -11,7 +11,16 @@ postgres_only do
 
     describe "#add_index" do
       # tested via adding index in migration
-      pending "add" do
+      it "should add a covering index if no type is specified" do
+        index_name = "contacts_age_index"
+        options = {
+          :type => nil,
+          :fields => [:age],
+          :order => {} of Symbol => Symbol,
+          :lengths => {} of Symbol => Symbol
+        }
+        adapter.add_index("contacts", index_name, options)
```

Comment:
do you mean postgres, this test shouldn't run for the mysql adapter as its got your handy postgres_only guard? Makes sense to insure its not there to begin with though. This will require updating the drop_index function for the postgres adapter.

Reply:
I've added a guard to delete the index if it exists to be on the safe side.

Reply:
I've missed that fact this is inside of `%postgres_only` :disappointed: